app/javascript/mastodon/features/keyboard_shortcuts/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Column from '../ui/components/column';
+import { defineMessages, injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import ImmutablePureComponent from 'react-immutable-pure-component';
+
+const messages = defineMessages({
+  heading: { id: 'keyboard_shortcuts.heading', defaultMessage: 'Keyboard Shortcuts' },
+  reply: { id: 'keyboard_shortcuts.reply', defaultMessage: 'to reply' },
+  mention: { id: 'keyboard_shortcuts.mention', defaultMessage: 'to mention author' },
+  favourite: { id: 'keyboard_shortcuts.favourite', defaultMessage: 'to favourite' },
+  boost: { id: 'keyboard_shortcuts.boost', defaultMessage: 'to boost' },
+  enter: { id: 'keyboard_shortcuts.enter', defaultMessage: 'to open status' },
+  profile: { id: 'keyboard_shortcuts.profile', defaultMessage: 'to open author\'s profile' },
+  up: { id: 'keyboard_shortcuts.up', defaultMessage: 'to move up in the list' },
+  down: { id: 'keyboard_shortcuts.down', defaultMessage: 'to move down in the list' },
+  column: { id: 'keyboard_shortcuts.column', defaultMessage: 'to focus a status in one of the columns' },
+  compose: { id: 'keyboard_shortcuts.compose', defaultMessage: 'to focus the compose textarea' },
+  toot: { id: 'keyboard_shortcuts.toot', defaultMessage: 'to start a brand new toot' },
+  back: { id: 'keyboard_shortcuts.back', defaultMessage: 'to navigate back' },
+  search: { id: 'keyboard_shortcuts.search', defaultMessage: 'to focus search' },
+  unfocus: { id: 'keyboard_shortcuts.unfocus', defaultMessage: 'to un-focus compose textarea/search' },
+  legend: { id: 'keyboard_shortcuts.legend', defaultMessage: 'to display this legend' },
+});
+
+@injectIntl
+export default class KeyboardShortcuts extends ImmutablePureComponent {
+
+  static propTypes = {
+    intl: PropTypes.object.isRequired,
+    multiColumn: PropTypes.bool,
+  };
+
+  render () {
+    const { intl } = this.props;
+
+    return (
+      <Column icon='question' heading={intl.formatMessage(messages.heading)} hideHeadingOnMobile>
+        <div className='keyboard-shortcuts scrollable optionally-scrollable'>
+          <table>
```

Comment:
I would add the header rows for accessibility reasons - we can give them `height: 0` if we decide to hide them for aesthetic reasons.

Reply:
@sorin-davidoi take a look at 2ec97c12f4b140a33a25f5f8cf4f9a8ed12eef07 ? I used a different CSS technique which should work for accessibility (had trouble getting `height: 0;` to work with the table, but if you want to suggest some CSS I'm happy to revise again).